src/extendables/User/Minion.ts

```diff
@@ -258,7 +258,7 @@ export default class extends Extendable {
 			case Activity.Fletching: {
 				const data = currentTask as FletchingActivityTaskOptions;
 				const fletchable = Fletching.Fletchables.find(
-					item => item.id === data.fletchableID
+					item => item.name === data.fletchableName
```

Comment:
since the activity already has the name, you dont need to find it in fletchable and can instead just straight use the name in the return message

Reply:
fixed